src/launchpad/parsers/apple/macho_parser.py

```diff
@@ -236,3 +241,39 @@ def has_swift_imageinfo(self) -> bool:
         except Exception as e:
             logger.error("Could not parse __objc_imageinfo: %s", e)
             return False
+
+    @trace(name="static_inits")
+    def static_inits(self) -> List[lief.Symbol | str]:
+        init_sec = self.get_section_bytes("__mod_init_func")
+        if not init_sec:
+            return []
+
+        ptr_size = 8 if self.binary.header.cpu_type == lief.MachO.Header.CPU_TYPE.ARM64 else 4
```

Comment:
This feels wrong but don't know the correct solutions right now. Do we need this check? And is the pointer size based on CPU or just whether it's 32/64 bit?

Reply:
good idea